functests/performance/hugepages.go

```diff
@@ -77,4 +84,100 @@ var _ = Describe("[performance]Hugepages", func() {
 	// 		Expect(string(nrHugepages)).To(Equal("128"))
 	// 	})
 	// })
+
+	Context("Basic hugepages usage verification", func() {
```

Comment:
also, can you add polarion test details, similar to CNV format - https://github.com/kubevirt/kubevirt/blob/9dd17aae95b027e8c7948d570d06aac173caea53/tests/console_test.go#L38